plugins/rtmp-services/data/services.json

```diff
@@ -1143,66 +1143,51 @@
-                    "url": "rtmp://ingest-us-west.b.switchboard.zone/live"
-                },
-                {
-                    "name": "US East 1 (North)",
-                    "url": "rtmp://ingest-us-east.a.switchboard.zone/live"
-                },
-                {
-                    "name": "US East 2 (South)",
-                    "url": "rtmp://ingest-us-east.b.switchboard.zone/live"
-                },
-                {
-                    "name": "US Central (North)",
-                    "url": "rtmp://ingest-us-central.a.switchboard.zone/live"
+                    "name": "US West",
+                    "url": "rtmp://ingest-us-west.switchboard.zone/live"
                 },
                 {
-                    "name": "South America East (São Paulo, BR)",
-                    "url": "rtmp://ingest-sa-east.a.switchboard.zone/live"
+                    "name": "US East",
+                    "url": "rtmp://ingest-us-east.switchboard.zone/live"
                 },
                 {
-                    "name": "Europe West (London, UK)",
-                    "url": "rtmp://ingest-eu-west.a.switchboard.zone/live"
+                    "name": "US Central",
+                    "url": "rtmp://ingest-us-central.switchboard.zone/live"
                 },
                 {
-                    "name": "Europe North (Hamina, FI)",
-                    "url": "rtmp://ingest-eu-north.a.switchboard.zone/live"
+                    "name": "South America",
+                    "url": "rtmp://ingest-sa.switchboard.zone/live"
                 },
                 {
-                    "name": "Australia Southeast (Sydney, AU)",
-                    "url": "rtmp://ingest-au-southeast.a.switchboard.zone/live"
+                    "name": "EU (auto)",
+                    "url": "rtmp://ingest-eu.switchboard.zone/live"
                 },
                 {
-                    "name": "Asia East (Changhua County, TW)",
-                    "url": "rtmp://ingest-as-east.a.switchboard.zone/live"
+                    "name": "Australia",
+                    "url": "rtmp://ingest-au.switchboard.zone/live"
                 },
                 {
-                    "name": "Asia Northeast (Tokyo, JP)",
-                    "url": "rtmp://ingest-as-northeast.a.switchboard.zone/live"
+                    "name": "APAC (auto)",
+                    "url": "rtmp://ingest-asia.switchboard.zone/live"
                 },
                 {
-                    "name": "Asia South (Mumbai, IN)",
-                    "url": "rtmp://ingest-as-south.a.switchboard.zone/live"
+                    "name": "UK",
+                    "url": "rtmp://ingest-uk.switchboard.zone/live"
                 }
             ],
             "recommended": {
                 "keyint": 2,
-                "max audio bitrate": 128,
-                "max video bitrate": 10000
-            }
+                "profile": "main",
```

Comment:
Are you sure this is wanted? It will reduce quality for compatibility with devices that are 10+ years old.

Reply:
What is the recommended setting for max compatibility?